docs/source/reference/tpu.rst

```diff
@@ -112,15 +190,18 @@ Now, we show a complete YAML for running `MNIST training <https://cloud.google.c
 
    TPU node requires loading data from a GCS bucket, so we add a :code:`file_mounts` to create a new bucket.
    Check :ref:`SkyPilot Storage <sky-storage>` for more details.
```

Comment:
```suggestion
   TPU node requires loading data from a GCS bucket. The :code:`file_mounts` spec above simplifies this by using :ref:`SkyPilot Storage <sky-storage>` to create a new bucket/mount an existing bucket.
   
```

Actually, is that creating a new one or mounting an old one?

Reply:
Either would work right? User can re-use an old one containing processed datasets or create a new one.